resources/jscomposition/cases/casesDetail/edit.js

```diff
@@ -1,8 +1,366 @@
 import Vue from "vue";
 import CaseDetail from "./components/CaseDetail.vue";
+import Timeline from "../../../js/components/Timeline.vue";
+
+Vue.component("Timeline", Timeline);
 
 new Vue({
   el: "#case-detail",
   components: { CaseDetail },
-  data() {},
+  data() {
+    return {
+      activeTab: "pending",
```

Comment:
Where do we use this variables in the template?
 Is necesary this variables in the new implementation?